src/platform/tests/TestConfigurationMgr.cpp

```diff
@@ -288,7 +288,7 @@ static void TestConfigurationMgr_PairingCode(nlTestSuite * inSuite, void * inCon
     NL_TEST_ASSERT(inSuite, err == CHIP_NO_ERROR);
 
     NL_TEST_ASSERT(inSuite, pairingCodeLen == strlen(pairingCode));
-    NL_TEST_ASSERT(inSuite, strcmp(buf, "016CB664A86A888D") == 0);
+    NL_TEST_ASSERT(inSuite, strcmp(buf, "16CB664A86A888D") == 0);
```

Comment:
Why isn't the hex representation a full 8 bytes anymore?

Reply:
./src/include/platform/ConfigurationManager.h:58:        kMaxPairingCodeLength      = 15

The maximum len of pairing code is defined to 15 in both CHIP and Weave. Should it be 16?

Reply:
After checking the logic, I believe we need to set the kMaxPairingCodeLength to 16 to accommodate  hex representation of 8 bytes.